android-agent/src/androidTest/kotlin/io/opentelemetry/android/BasicInstrumentationTest.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android
+
+import android.app.Application
+import android.os.Build.VERSION_CODES.LOLLIPOP
+import android.os.Build.VERSION_CODES.UPSIDE_DOWN_CAKE
+import androidx.test.core.app.ApplicationProvider.getApplicationContext
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import io.opentelemetry.android.config.OtelRumConfig
+import org.junit.Assert.assertNotNull
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+
+@RunWith(AndroidJUnit4::class)
+internal class BasicInstrumentationTest {
+    @Config(sdk = [LOLLIPOP, UPSIDE_DOWN_CAKE])
+    fun startOpenTelemetryRumInAndroid() {
```

Comment:
We think `@Test` is missing.
```suggestion
    @Test
    fun startOpenTelemetryRumInAndroid() {
```